records_mover/records/targets/table/move_from_temp_loc_after_filling_it.py

```diff
@@ -28,7 +28,9 @@ def __init__(self,
     @contextmanager
     def temporary_loadable_directory_loc(self) -> Iterator[BaseDirectoryUrl]:
         driver = self.tbl.db_driver(self.tbl.db_engine)
-        with driver.temporary_loadable_directory_loc() as loc:
+        loader = driver.loader()
+        # TODO: Should force null check
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker